vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -644,9 +645,32 @@ void WinRTHttpResource::AddResponseHandler(shared_ptr<IResponseHandler> response
 /*static*/ shared_ptr<IHttpResource> IHttpResource::Make(
     winrt::Windows::Foundation::IInspectable const &inspectableProperties) noexcept {
   using namespace winrt::Microsoft::ReactNative;
+  using namespace winrt::Windows::Web::Http::Filters;
   using winrt::Windows::Web::Http::HttpClient;
 
-  auto redirFilter = winrt::make<RedirectHttpFilter>();
+  auto getPropertyBag = [&]() {
+    if (!inspectableProperties) {
+      return ReactPropertyBag{};
+    }
+    return ReactPropertyBag{inspectableProperties.try_as<IReactPropertyBag>()};
+  };
+
+
+
+  IHttpFilter customFilter{nullptr};
+  if (auto propBag = getPropertyBag()) {
+    if (auto factoryDelegate = GetHttpFilterFactoryDelegate(propBag)) {
+      customFilter = factoryDelegate();
+    }
+  }
+
+  IHttpFilter redirFilter{nullptr};
+  if (customFilter) {
+    redirFilter = winrt::make<RedirectHttpFilter>(std::move(customFilter), HttpBaseProtocolFilter{});
```

Comment:
Please instantiate two custom filters to pass into this constructor.
i.e.
```cpp
redirFilter = winrt::make<RedirectHttpFilter>(std::move(customFilter1), std::move(customFilter2));
```

This is not clearly explained in the `RedirectHttpFilter` sources, but we use two separate internal filters to handle requests with/without credentials.